kiwix-zim.sh

```diff
@@ -137,12 +142,16 @@ usage_example() {
     echo '    /full/path/                Full path to ZIM directory'
     echo
     echo 'Options:'
+    echo '    -c, --calculate-checksum   Verifies that the downloaded files were not corrupted, but can take a while for large downloads.'
     echo '    -d, --disable-dry-run      Dry-Run Override.'
     echo '                               *** Caution ***'
     echo
     echo '    -h, --help                 Show this usage and exit.'
-    echo
     echo '    -p, --skip-purge           Skips purging any replaced ZIMs.'
+    echo '    -n <size>, --min-size      Minimum ZIM Size to be downloaded.'
+    echo '                               Specify units include M Mi G Gi, etc. See `man numfmt`'
+    echo '    -x <size> , --man-size     Maximum ZIM Size to be downloaded.'
```

Comment:
@jojo2357 Should this be `--max-size` rather than `--man-size`?

Reply:
Yes 😂